ldoc/html/ldoc_ltp.lua

```diff
@@ -226,7 +227,9 @@ return [==[
 #  if show_return and item.retgroups then local groups = item.retgroups
     <h3>Returns:</h3>
 #   for i,group in ldoc.ipairs(groups) do local li,il = use_li(group)
+#   if #group > 1 then
```

Comment:
The nesting of these if loops should be reversed.

Reply:
That is outside the scope. This was only to get the generated html pass a linter.